setup.py

```diff
@@ -60,16 +61,28 @@
     compiler_directives['linetrace'] = True
     define_macros.append(("CYTHON_TRACE_NOGIL", "1"))
 
+
+if sys.platform == "darwin":
+    os.environ["CC"] = "/usr/local/opt/llvm/bin/clang -L/usr/local/opt/llvm/lib -Wl,-rpath,/usr/local/opt/llvm/lib"
+    os.environ["CXX"] = "/usr/local/opt/llvm/bin/clang++ -L/usr/local/opt/llvm/lib -Wl,-rpath,/usr/local/opt/llvm/lib"
+
 GEODESIC_NAME = "gdist"
 
 GEODESIC_MODULE = [
     setuptools.Extension(
         name=GEODESIC_NAME,  # Name of extension
         sources=["gdist.pyx"],  # Filename of Cython source
         language="c++",  # Cython create C++ source
-        define_macros=define_macros,
-        extra_compile_args=['--std=c++14'],
-        extra_link_args=['--std=c++14'],
+        # Disable assertions; one is failing geodesic_mesh.h:405
+        define_macros=[('NDEBUG', 1)],
```

Comment:
I think line 77 is breaking the coverage configuration (probably in the merge the wrong choice was left).

Reply:
Ah, yes! 🤦‍♂️ Thanks!